ts/src/pro/binance.ts

```diff
@@ -2998,7 +2998,7 @@ export default class binance extends binanceRest {
             }
         }
         if (!rejected) {
-            client.reject (message, id);
+            client.reject (message, id); // todo: needs revision, first arg might need to be an exception
```

Comment:
@carlosmiei for all other exchanges, i've turned the string into Exception type, however, for binance, i've refrained to edit this, idk, left for your revision